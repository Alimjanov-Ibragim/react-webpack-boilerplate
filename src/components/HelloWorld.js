import React, {Component} from 'react'

export default class HelloWorld extends Component {
  plus(a,b) {
    return a+b;
  }

  render() {
    return (
      <div>
        <h1>Hello World</h1>
      </div>
    )
  }
}
