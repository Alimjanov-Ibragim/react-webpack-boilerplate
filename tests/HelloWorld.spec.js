import React from 'react'
import renderer from 'react-test-renderer'
import HelloWorld from '../src/components/HelloWorld'

describe('function summa', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <HelloWorld />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('sum 1+2=3', () => {
    const hello = new HelloWorld;
    expect(hello.plus(1,2)).toEqual(3);
  });
})
