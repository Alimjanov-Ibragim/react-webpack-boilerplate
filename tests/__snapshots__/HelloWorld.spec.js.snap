// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`function summa renders correctly 1`] = `
<div>
  <h1>
    Hello World
  </h1>
</div>
`;
